Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Multiplayer Samples Co-op Changelog
 
+## [unreleased] - yyyy-mm-dd
+
+### Changed
+* Upgraded Utilities package to Netcode for GameObjects v1.8.1 (#883)
```

Comment:
Do we need to add the other changes from the changelogs too?

* Upgraded to the newer API for Rpcs, Universal Rpcs
* ClientConnectedState has been modified to account for server/host now populating DisconnectReason before disconnecting a client before shutting down

Reply:
Good catch, I'll add the note about Universal Rpcs and the fact that the namespace had to be updated to include Unity.Collections.
The last point is part of the project's changelog, so that seems like it should live only there.

Reply:
Awesome!